code/game/objects/structures/extinguisher.dm

```diff
@@ -68,7 +68,7 @@
 	return
 
 /obj/structure/extinguisher_cabinet/AltClick(mob/living/user)
-	if(user.incapacitated() || !Adjacent(user))
+	if(user.incapacitated() || !Adjacent(user) || !ishuman(user))

```

Comment:
```isliving()``` ans the other ```isX()``` we've defined were changed to macros a while ago, so it's the difference between ```istype(X, /mob/living)``` and ```istype(X)``` I'm not sure which byond performs better in (if at all), explicit or implicit types.